plugin/storage/integration/integration.go

```diff
@@ -538,3 +538,12 @@ func (s *StorageIntegration) RunAll(t *testing.T) {
 	t.Run("GetThroughput", s.testGetThroughput)
 	t.Run("GetLatestProbability", s.testGetLatestProbability)
 }
+
+// RunTestSpanstore runs only span related integration tests
+func (s *StorageIntegration) RunTestSpanstore(t *testing.T) {
```

Comment:
ok, but invoke it from RunAll instead of repeating
```suggestion
func (s *StorageIntegration) RunSpanStoreTests(t *testing.T) {
```